### PR TITLE
Login script

### PR DESCRIPTION
**Pull request by bartorl**

### CODE REVIEW

File: views/auth/login.ejs [hunk 1]
```diff
@@ -24,7 +24,39 @@
...
+    const formData = new FormData(loginForm);
+
+
+    request.addEventListener('load', (event) => {
+        const response = JSON.parse(event.target.responseText);
+
+        if(response.redirect === 'profile'){
+            window.location.href = '/profile';
+        } else {
+            if(response.msg !== null){
```
Comment (macibol775) [inline views/auth/login.ejs:50]: can be replaced with else if(response.msg)

**Review by macibol775 — APPROVED** [review]
if works ok

File: controllers/authController.js [hunk 2]
```diff
@@ -40,14 +40,17 @@ const login_post = async (req, res) => {
         try {
             const applicant = await Applicant.login(email, password);
             if (applicant) {
-                console.log('OK');
+                console.log("OK");
```
Comment (MasterCoookie) [inline controllers/authController.js:43]: not needed log

File: controllers/authController.js [hunk 3]
```diff
@@ -40,14 +40,17 @@ const login_post = async (req, res) => {
...
+                res.status(202).json({ redirect: 'profile' });
             } else {
-                res.status(403).json({ msg: 'Invalid credentials' });
+                res.json({ msg: 'Niewłaściwe dane' });
```
Comment (MasterCoookie) [inline controllers/authController.js:46]: no code

File: controllers/authController.js [hunk 4]
```diff
@@ -40,14 +40,17 @@ const login_post = async (req, res) => {
...
             }
         } catch (e) {
-            console.log(e);
-
+            //res.json(e);
+            res.json({ msg: 'Niewłaściwe dane' });
```
Comment (MasterCoookie) [inline controllers/authController.js:50]: no code

**Review by MasterCoookie — CHANGES_REQUESTED** [review]
add code

**Review by MasterCoookie — APPROVED** [review]

**Review by WojciechPtas — COMMENTED** [review]
LGTM

**Pull request merged**